finals-2024-01-16/code/5140-simona-blazevska/src/app/band-detail/band-detail.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Band } from '../models/bands';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-band-detail',
+  templateUrl: './band-detail.component.html',
+  styleUrls: ['./band-detail.component.css'],
+})
+export class BandDetailComponent implements OnInit {
+  band: Band | undefined;
+  yearsActive: string | undefined;
+  similarBands: Band[] = [];
+
+  constructor(private route: ActivatedRoute, private http: HttpClient) {}
```

Comment:
+5 points for injecting and using `ActivatedRoute`
+5 points for correct `HttpClient` usage